xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -31,18 +31,18 @@ class CGUIDialogVideoInfo :
 public:
   CGUIDialogVideoInfo(void);
   virtual ~CGUIDialogVideoInfo(void);
-  virtual bool OnMessage(CGUIMessage& message);
-  virtual bool OnAction(const CAction &action);
+  bool OnMessage(CGUIMessage& message) override;
+  bool OnAction(const CAction &action) override;
```

Comment:
I understand both sides, when not using an ide it's easier to spot the virtual. When using ReSharper or Clion it's distracting seeing it marked as unnecessary code, I lean towards getting rid of it though